build.gradle

```diff
@@ -90,23 +104,28 @@ allprojects {
   }
 
   task smoke(type: Test, description: 'Runs the smoke tests.', group: 'Verification') {
-    setTestClassesDirs(sourceSets.integrationTest.output.classesDirs)
-    setClasspath(sourceSets.integrationTest.runtimeClasspath)
+    setTestClassesDirs(sourceSets.functionalTest.output.classesDirs)
+    setClasspath(sourceSets.functionalTest.runtimeClasspath)
 
     useJUnit {
-      includeCategories 'uk.gov.hmcts.reform.pdf.service.integration.SmokeTest'
+      includeCategories 'uk.gov.hmcts.reform.pdf.service.SmokeTest'
     }
```

Comment:
SmokeTest category is removed as part of this pr so do we need this ?

Reply:
moved to functional tests not removed